.gitignore

```diff
@@ -1,2 +1,5 @@
 ipmi_exporter
 *.tar.gz
+
+# Vscode files
+.vscode
```

Comment:
Instead of adding every individual's editor files to project ignore files, please use a global gitignore file for yourself. See https://gist.github.com/subfuzion/db7f57fff2fb6998a16c as an example (first google result, there are more docs).